src/main/java/data/exception/DukeException.java

```diff
@@ -0,0 +1,40 @@
+package data.exception;
+
+/**
+ * The custom {@link Exception} class for Duke.
+ * Used by {@link ui.Ui} to print error messages.
+ */
+public class DukeException extends Exception {
```

Comment:
Can the exception name be more specific and coherent?

Reply:
I wrote this class to handle all Duke-related exceptions so I don't really know how to make the name more specific. Would something like `DukeErrorException` or `DukeParseException` be better?